microhapulator/data/fancyTable.js

```diff
@@ -0,0 +1,269 @@
+/*!
```

Comment:
We'll want to add the relative path to this file to `.gitattributes` and mark it as `linguist-vendored`. This indicates that it's third-party code and will exclude it from e.g. language repo statistics.

Reply:
done